### PR TITLE
feat(formula): store macros.jinja in subdir, trigger major release

### PR DESCRIPTION
**Pull request by noelmcloughlin**
### PR progress checklist (to be filled in by reviewers)

- [x] Changes to documentation are appropriate (or tick if not required)
- [x] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [x] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Related issues and/or pull requests

Mark #48 as breaking change so the major release is triggered.

And close: #43 

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

File: node/package/repo/install.sls [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {#- Get the `tplroot` from `tpldir` #}
 {%- set tplroot = tpldir.split('/')[0] %}
 {%- from tplroot ~ "/map.jinja" import node with context %}
-{%- from tplroot ~ "/macros.jinja" import format_kwargs with context %}
+{%- from tplroot ~ "/macros/macros.jinja" import format_kwargs with context %}
```
Comment (myii) [inline node/package/repo/install.sls:7]: This PR's changeset shows `node/macros.jinja → node/files/macros.jinja`, so `"/macros/macros.jinja"` would have to be fixed accordingly.

File: node/source/install.sls [hunk 2]
```diff
@@ -4,7 +4,7 @@
 {#- Get the `tplroot` from `tpldir` #}
 {%- set tplroot = tpldir.split('/')[0] %}
 {%- from tplroot ~ "/map.jinja" import node with context %}
-{%- from tplroot ~ "/macros.jinja" import format_kwargs with context %}
+{%- from tplroot ~ "/macros/macros.jinja" import format_kwargs with context %}
```
Comment (myii) [inline node/source/install.sls:7]: This PR's changeset shows `node/macros.jinja → node/files/macros.jinja`, so `"/macros/macros.jinja"` would have to be fixed accordingly.

File: node/package/repo/install.sls (hunk 1 above)
Comment (noelmcloughlin) [reply node/package/repo/install.sls:7]: Thanks, not sure why I missed that. Sloppy.

File: node/source/install.sls (hunk 2 above)
Comment (noelmcloughlin) [reply node/source/install.sls:7]: Done thanks @myii

**Comment by noelmcloughlin** [comment]
That's done @myii  thanks for reviewing and patience getting this completed.

**Comment by noelmcloughlin** [comment]
Travis CI is passed and I fixed conflicts with master again

**Comment by noelmcloughlin** [comment]
@myii can this be merged.

**Comment by myii** [comment]
Thanks, @noelmcloughlin.

**Pull request merged**